boot/flags_test.go

```diff
@@ -365,6 +365,8 @@ func (s *bootFlagsSuite) TestUserspaceBootFlagsUC20(c *C) {
 }
 
 func (s *bootFlagsSuite) TestRunModeRootfs(c *C) {
```

Comment:
don't we need new tests?

Reply:
That is spot on! We need new tests and the new tests showed a mishandling on classic. I fixed that now.